### PR TITLE
Implement fade-in on viewport at `/wordpress`, `/hub` & `/homepage`

### PR DESCRIPTION
**Pull request by yusufkinatas**
## 🌟 What is the purpose of this PR?

This PR implements a fade-in animation on main elements of home page, /wordpress and /hub pages. So elements fade-in once when they are visible to the user (shown at viewport). 

Also to keep the page loading time shorter, implemented lazy loading for `framer-motion`. They explain using a lazy loading version of the `motion` component on their [official docs](https://www.framer.com/motion/guide-reduce-bundle-size/) to reducing the bundle size.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1204000740778935/1204162407205361/f) _(internal)_
- [Asana task](https://app.asana.com/0/1204000740778935/1204165363163231/f) _(internal)_

## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Confirm that /hub, /wordpress & home pages have fade-in-on-scroll animation on elements.

## 📦 Lazy Loading - Bundle Size Demo

Check [this commit](https://github.com/blockprotocol/blockprotocol/pull/1203/commits/8cf004e8bed18563ba0197ce23cc2ba9a8ae8ab9) to see lazy-loading implementation separately

|             |             |             |
|:-------------------------------------:|:-------------------------------------:|:-------------------------------------:|
|Build without `framer-motion`|Build with `framer-motion`                                       | Build with lazy `framer-motion`                                      |
|  app.js = 353 kB | app.js = 383 kB  | app.js = 358 kB  |
|   |   | framer-features = 16.1 kB |

## 📹 Demo

https://user-images.githubusercontent.com/39553853/226611173-4e18ae55-738d-4069-8ef6-dda27bbbea05.mov

### CODE REVIEW

**Comment by yusufkinatas** [comment]
@nonparibus Please confirm if the animations are looking as intended.

**Comment by nonparibus** [comment]
@yusufkinatas Did you consciously omit above-fold animation, for reasons related due to lazy-loading or other?

File: apps/site/.eslintrc.cjs [hunk 1]
```diff
@@ -29,6 +29,11 @@ module.exports = {
             importNames: ["FC", "VFC", "VoidFunctionComponent"],
             message: "Please use FunctionComponent instead.",
           },
+          {
+            name: "framer-motion",
+            importNames: ["motion"],
+            message: "Please use 'm' instead.",
```
Comment (CiaranMn) [inline apps/site/.eslintrc.cjs:35]: Can we explain why please?

Comment (yusufkinatas) [reply apps/site/.eslintrc.cjs]: Updated it as `Please use 'm' instead for 'framer-motion' to lazy-load correctly.`

**Review by CiaranMn — APPROVED** [review]
Looks good!

**Pull request merged**